To-do-list/Model/TaskManager.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  TaskManager.swift
+//  To-do-list
+//
+//  Created by ineta.magone on 03/03/2022.
+//
+
+import UIKit
+
+class TaskManager {
+    
+    // MARK: - Variables
+    
+    var titleArray = [String]()
+    var descriptionArray = [String]()
```

Comment:
let's make a struct Task {
var title: String
var description: String
}

and save tasks: [Task] = []

Reply:
Ok, thank You, will add that